frame/fnft/src/test/nonfungibles/mutate.rs

```diff
@@ -46,6 +50,20 @@ mod mint_into {
 				(ALICE, BTreeMap::new()),
 				"owner should be ALICE, with no attributes"
 			);
+			// get the asset account of the fNFT
+			let asset_account =
+				Pallet::<MockRuntime>::asset_account(&TEST_COLLECTION_ID, &NEW_NFT_ID);
+			// check the proxies
+			assert_ok!(pallet_account_proxy::Pallet::<MockRuntime>::find_proxy(
```

Comment:
You can just use `AccountProxy` as defined by the `construct_runtime!` macro

Reply:
Same for the uses of `Pallet::<MockRuntime>`, it can just be `StakingRewards` (or whatever it is this pallet is called)

Reply:
Done